### PR TITLE
console: Allow setting NodePort

### PR DESCRIPTION
**Pull request by RafalKorepta**

### CODE REVIEW

File: charts/console/templates/service.yaml [hunk 1]
```diff
@@ -30,4 +30,5 @@ spec:
       targetPort: {{ include "console.containerPort" . }}
       protocol: TCP
       name: http
+      nodePort: {{ .Values.service.nodePort }}
```
Comment (alejandroEsc) [inline charts/console/templates/service.yaml:33]: Should we gate these if the service.type is nodeport?

Comment (alejandroEsc) [reply charts/console/templates/service.yaml:33]: Also we have service files depending on type in redpanda, should we follow the same approach here?

**Review by alejandroEsc — COMMENTED** [review]
a few questions.

File: charts/console/templates/service.yaml (hunk 1 above)
Comment (RafalKorepta) [reply charts/console/templates/service.yaml:33]: Sure I will update this PR

Comment (RafalKorepta) [reply charts/console/templates/service.yaml:33]: > Also we have service files depending on type in redpanda, should we follow the same approach here?

We have only NOTES.txt that depend on type of the service. This is console chart, so I'm not sure how Redpanda chart applies here?

Comment (alejandroEsc) [reply charts/console/templates/service.yaml:34]: It doesn't, but in that chart we have two different service files, one per each type, just wondering if you think we need to do this here.

Comment (RafalKorepta) [reply charts/console/templates/service.yaml:34]: > It doesn't, but in that chart we have two different service files, one per each type, just wondering if you think we need to do this here.

Can you send link to Github where the service file is defined?

Comment (alejandroEsc) [reply charts/console/templates/service.yaml:34]: https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/templates/service.loadbalancer.yaml and
https://github.com/redpanda-data/helm-charts/blob/main/charts/redpanda/templates/services.nodeport.yaml

**Review by alejandroEsc — APPROVED** [review]
I wasnt gating this PR. Just asking a question, lgtm.

File: charts/console/templates/service.yaml (hunk 1 above)
Comment (RafalKorepta) [reply charts/console/templates/service.yaml:34]: We agree that changes to console can not be guarded by higher level charts. If console would depend on Redpanda then we could create some helper function.

**Comment by RafalKorepta** [comment]
I opened new PR as CLA boot doesn't report status and I'm unable to merge this PR

**Pull request merged**